osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -230,8 +231,14 @@ private void load(OsuGameBase game, OsuColour colours, AudioManager audio)
 
         protected override void LoadComplete()
         {
+            SelectedMods.BindValueChanged(_ => Scheduler.AddOnce(recreateContent));
+
             // this is called before base call so that the mod state is populated early, and the transition in `PopIn()` can play out properly.
-            globalAvailableMods.BindValueChanged(_ => createLocalMods(), true);
+            globalAvailableMods.BindValueChanged(_ =>
+            {
+                createLocalMods();
+                Scheduler.AddOnce(recreateContent);
+            }, true);
```

Comment:
Are the schedules doing anything else than debouncing here? I'd hope scheduling is not part of the fix...

Reply:
Just to avoid multiple calls on the same frame, yeah. Can be removed if you think it's ugly.